onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -827,6 +827,13 @@ std::unique_ptr<IExecutionProvider> CreateExecutionProviderInstance(
           } else if (option.first == "trt_op_types_to_exclude") {
             trt_op_types_to_exclude = option.second;
             params.trt_op_types_to_exclude = trt_op_types_to_exclude.c_str();
+          } else if (option.first == "trt_preview_features") {
+            if (!option.second.empty()) {
+              preview_features = option.second;
+              params.trt_preview_features = preview_features.c_str();
+            } else {
+              ORT_THROW("[ERROR] [TensorRT] The value for the key 'trt_preview_features' should be a semi-colon seperated string i.e. 'preview_features'.\n");
```

Comment:
nit: this throw seems not needed. Basically, when it is empty, it has no effect.